server/bin/pbench-verify-indexed.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/libexec/platform-python
+# -*- mode: python -*-
+
+"""Pbench Re-Index
```

Comment:
I guess we can tell where this code started ... but maybe you should update the docstring. 😆